reset.c

```diff
@@ -0,0 +1,888 @@
+        return 0;               //reject all
+    //else reject the selected ones
+
+    if (pgrejname) {
+        //remove explicit reject requests
+        pgfile = fopen(pgrejname, "r");
+        if (pgfile) {
+            if (parsecsvfile(pgfile, getcsvpgnames, config)) {
+                ret = 1;
+                printf("Failed to parse PGs csv file!\n");
+            }
+            fclose(pgfile);
+            pgfile = NULL;
+        }
+        else {
+            printf ("Could not open pg reject file!\n");
+            return 1;
+        }
+    }
+    return ret;
+}
+
+/// getauxtags - parse and get the aux tags from csv files and add to internal set; adds default tags and tags from keep file and remove those given in reject file
+/** @param config - pointer to conf_data which holds internal configuration and accepted aux tag set
+ *  @param auxkeepname - name of csv file containing aux tags which are to be retained
+ *  @param auxrejname - name of csv file containing aux tags which are to be rejected
+returns 1 on failure and 0 on success
+*/
+int getauxtags(conf_data *config, const char *auxkeepname, const char *auxrejname)
+{
+    int ret = 0;
+    FILE *tagfile = NULL;
+
+    if (!config) {
+        printf("Invalid parameter in getauxtags!\n");
+        return 1;
+    }
+
+    //add default tags
+    if (getdefaultauxtags(config))
+        return 1;
+
+    if (auxkeepname) {
+        //add explicit keep requests
+        tagfile = fopen(auxkeepname, "r");
+        if (tagfile) {
+            config->tmpfield = 0;           //0 to indicate keep tag - add to set
+            ret = parsecsvfile(tagfile, getcsvauxtags, config);
+            fclose(tagfile);
+            tagfile = NULL;
+            if (ret) {
+                printf("Failed to parse aux keep csv file!\n");
+                return 1;
+            }
+        }
+        else {
+            printf("Could not open aux keep file!\n");
+            return 1;
+        }
+    }
+
+    if (auxrejname) {
+        //remove explicit reject requests
+        tagfile = fopen(auxrejname, "r");
+        if (tagfile) {
+            config->tmpfield = 1;           //1 to indicate reject tag - remove from set
+            ret = parsecsvfile(tagfile, getcsvauxtags, config);
+            fclose(tagfile);
+            tagfile = NULL;
+            if (ret) {
+                printf("Failed to parse aux reject csv file!\n");
+                return 1;
+            }
+        }
+        else {
+            printf ("Could not open aux reject file!\n");
+            return 1;
+        }
+    }
+
+    if (!config->keepRGs) {
+        //RG lines not present, no point in having RG aux fields either, remove RG
+        khiter_t iter = kh_get(aux_set, config->auxtagsin, TAGNUM("RG"));
+        iter != kh_end(config->auxtagsin) ? kh_del(aux_set, config->auxtagsin, iter) : 0;
+    }
+
+    return 0;
+}
+
+int __get_aux_data(uint8_t* data, uint8_t *end, char *tag, char *type, void** ptr, int* len)
```

Comment:
What is this?  It looks unused.

Also generally we try to avoid starting functions with underscores as it's a namespace reserved for system use (although the chances of a clash are miniscule, so it's more of a style thing than something that's liable to cause breakage).

Reply:
Removed - missed during cleanup!.